### DISCUSSION TITLE
obs stream tools not working

### DISCUSSION THREAD

**Discussion by vamps316** [elgatosf/elgato#23] [Help]
i have the stream deck xl, everything is all fine until i stream once then after i close everything and go to bed or just finish streaming, all of a sudden the stream deck obs buttons aren't working, i found the solution to be to uninstall then reinstall the program but i have to do that every time. i tried just repairing the program but after the recent update, its stopped working. i get it if this happened every so often but every time? and its just the obs buttons, the most i can do is open the program. is it because my computer is not all 4k? do i need to turn something off? if there is an actual solution can someone make or link a video? i want to get my money's worth out of this.

**Comment by ChekTek** [comment]
Hi!

For plugin support, please use the support link found in the plugin's Stream Deck store page. For more general Elgato support, you can reach out on https://www.reddit.com/r/elgato/ or http://e.lga.to/help.
